Dockerfile

```diff
@@ -7,13 +7,13 @@ ENV YOUR_ENV=${YOUR_ENV} \
     PYTHONUNBUFFERED=1 \
     PIP_NO_CACHE_DIR=off \
     PIP_DISABLE_PIP_VERSION_CHECK=on \
-    POETRY_VERSION=1.0.0
+    POETRY_VERSION=1.0.0 \
+    POETRY_VIRTUALENVS_CREATE=false
```

Comment:
tell poetry not to use virtual envs (as dependencies are already isolated inside the docker container). This seems to be the recommended way to do this according to https://github.com/python-poetry/poetry/issues/1265#issuecomment-543717607